### ISSUE TITLE
Как он работает в OpenWRT?

### ISSUE THREAD

**Issue by dcbeelinekz2** [janvidar/uhub#63]
**users.conf**

```
#operator "Ник" "Пароль"
#user "Ник" "Пароль"
#admin "Ник" "Пароль"
admin "DCBEELINEKZ" "RAFA"
```

ERROR: Unknown ACL command on line 4: 'admin "DCBEELINEKZ" "RAFA"'

**Comment by Kcchouette** [comment]
**users.conf** example: https://github.com/janvidar/uhub/blob/master/doc/users.conf  
seems using `user_admin` instead of just `admin`

**Comment by dcbeelinekz2** [comment]
И что за ошибки 
root@LEDE:~# uhub -L
2019-02-05 03:36:56  ERROR: net_bind, fd=6: Address in use (125)
2019-02-05 03:36:56  ERROR: hub_start_service(): Unable to bind to TCP local address. errno=22, str=Invalid argument
2019-02-05 03:36:56  FATAL: Unable to start hub service

**Comment by dcbeelinekz2** [comment]
> **users.conf** example: https://github.com/janvidar/uhub/blob/master/doc/users.conf
> seems using `user_admin` instead of just `admin`

нет, пробовал разные варианты, не получается в OpenWT с этим файлом

**Comment by Tantrix** [comment]
ACL ≠ users.conf

**Comment by vkellrafa-ux** [comment]
Новая версия есть исходники makefile?

**Comment by janvidar** [comment]
There are two ways forward;
1. Use CMake
2. Use Zig

The old Makefile is not used any more.
